content/accessibility/links.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Links
+---
+
+## Overview
+Links are user interface elements that *navigate* you to a new place or new content. Contrast this with buttons, which are designed to *activate* a feature.
+
+## Why?
+Links can do things like help with page context, reference similar items of interest, and allow for endless connected information surfing through sites like Wikipedia. Links play a key part in your experience on the web, but without proper consideration those links can be frustrating, skipped over, or completely unnoticed. 
+
+For screen reader assistive technology, links and buttons are expected to function differently. If a link is activated and does not do what was expected, that can be disorienting and frustrating. 
+
+A common way a screen reader might navigate the page is by going through a list of all the links on the page. Without context, "read more" or "click here" links won't be very helpful. 
+
+People who have low or colorblind vision may have trouble identifying links that just use color to distinguish between plain text, this is why keeping the underline styling on links within body text is important for identification.
+
+## How to test links
+- Check a link's function. If clicking a link instead activates a feature on the page, it should be a `<button>`, and vice versa. 
+- Check a link's purpose. If you can't get an idea of where a link will take you based on the link text alone, the link text needs updates. 
+- [Check a link's contrast](https://webaim.org/resources/contrastchecker/). Links being text supersede the widget contrast requirement, which means all links need to pass 4.5:1 against the background color that they are on. 
```

Comment:
```suggestion
- [Check a link's contrast](https://webaim.org/resources/contrastchecker/). Because links are text-based, they need to pass 4.5:1 against the background color that they are on.
```